packages/documentation/src/stories/components/rating/post-rating.docs.mdx

```diff
@@ -0,0 +1,49 @@
+import { Canvas, Controls, Meta} from '@storybook/blocks';
+import LinkTo from '@storybook/addon-links/react';
+import * as RatingStories from './post-rating.stories';
+
+<Meta of={RatingStories} />
+
+# Rating
+
+Our Rating Web Component allows you to easily integrate 
+a customizable star rating system, providing an 
+interactive and visually appealing way to gather and display user feedback.
+
+<Canvas of={RatingStories.Default} sourceState='shown'/>
+<div className="hide-col-default">
+  <Controls of={RatingStories.Default} />
+</div>
+
+## Installation
+
+The `<post-rating/>` element is part of the `@swisspost/design-system-components` package.
+For more information, read the <LinkTo kind="getting-started-components" story="docs">getting started with components guide</LinkTo>.
+
+## Examples
+
+### Disabled
+
+Is the disabled property set, the component has a clear disabled style, 
+so the user can still see the current rating but is not able to interact with the component.
+  
+
+<Canvas of={RatingStories.Disabled} />
+<div className="hide-col-default">
+  <Controls of={RatingStories.Disabled} />
+</div>
+
+### Readonly
+
+Is the readonly property set, is the component not interactive
+but the current rating is still visible as in the default state.
+
+<Canvas of={RatingStories.Readonly} />
+<div className="hide-col-default">
+  <Controls of={RatingStories.Readonly} />
+</div>
```

Comment:
This is already explained and handled in the default story. If at all, I would add examples, which show how to event listener to the given events. Otherwise, I would remove this duplicated examples.